### ISSUE TITLE
Skyrider placed randomly (?) in Drassen Airport

### ISSUE THREAD

**Issue by JAsmine-ja2** [ja2-stracciatella/ja2-stracciatella#478]
Sometimes, Skyrider-NPC is placed seemingly random on B13. Isn't he supposed to stay near his helicopter?
Having him hang around somewhere else is inconvenient and potentially dangerous for him. He would be much safer near his heli.
![skyrider](https://cloud.githubusercontent.com/assets/17213795/21655428/482ffba8-d2ba-11e6-94c3-88cd72cb8a34.jpg)

Savedgame:
[SaveGame10.zip](https://github.com/ja2-stracciatella/ja2-stracciatella/files/685626/SaveGame10.zip)

**Comment by lynxlynxlynx** [comment]
Can't NPCs wander?

**Comment by JAsmine-ja2** [comment]
Yes, they can. If and how they can wander is specified in their 'orders' in the map editor.
The basic orders are:
Stationary: stays on the same tile unless under fire - if so, he can move a few (5?) tiles away to seek cover
On Guard: patrols a 5-tile radius around his tile
Close Patrol: patrols a 15-tile radius around his tile
Far Patrol: patrols a 25-tile radius around his tile - can increase if the AI is in red (?) state

Skyriders settings (Vanilla JA2 Gold 1.12 English) are:
![skyrider-stationary](https://cloud.githubusercontent.com/assets/17213795/21686491/e9bcc7e8-d365-11e6-880c-3892e6735e15.jpg)

Stationary. He should always be next to his heli.

Maybe, Skyrider is misplaced when/after the player uses the heli?
